models/maisi_ct_generative/configs/metadata.json

```diff
@@ -1,7 +1,8 @@
 {
     "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_generator_ldm_20240318.json",
-    "version": "0.4.5",
+    "version": "0.4.6",
     "changelog": {
+        "0.4.6": "add TensorRT support",
```

Comment:
HI @binliunls , this bundle is still use monai 1.4.0, do we need a new version here to support TRT?

Reply:
Can we use the dev branch for this?

Reply:
no, we need a tag. CI will create a venv based on the metadata.json

Reply:
@KumoLiu  could you please help to tag a RC version of MONAI core including [the TRT PR](https://github.com/Project-MONAI/MONAI/pull/8153) so that we can merge this one to model zoo?

Thanks in advance,  
Bin

Reply:
@yiheng-wang-nv @binliunls, 1.4.1rc1 uploaded. https://pypi.org/project/monai/#history